charts/incubator/airdcpp-webclient/values.yaml

```
@@ -0,0 +1,38 @@
+image:
+  repository: tccr.io/truecharts/airdcpp-webclient
+  pullPolicy: IfNotPresent
+  tag: v2.11.2@sha256:2b3eb50f1bdc537d887bacee6e815cb0b4fb6c53bac723b3267f3c4d13a22274
+
+service:
+  main:
+    ports:
+      main:
+        port: 10187
```

```suggestion
      main:
        port: 10155
```
I think you meant `10087`, but `10087` is used in core apps